sympy/core/tests/test_numbers.py

```diff
@@ -1917,3 +1917,10 @@ def test_NegativeInfinity():
     assert((-oo).ceiling() == -oo)
     assert((-oo)**11 == -oo)
     assert((-oo)**12 == oo)
+
+def test_issue_6133():
+    raises(TypeError, lambda: (-oo < None))
+    raises(TypeError, lambda: (-2 < None))
```

Comment:
This and the case with `2` below are not really testing any part of SymPy since `-2` is a regular Python int and `None` is a Python builtin. I think those two tests should use e.g. `S(-2)` so we're testing that this is what happens with sympy's `Integer`s.

Reply:
thanks, updated it please review.